src/entities/card/organisms/card-preview.tsx

```diff
@@ -35,15 +37,29 @@ interface CardPreviewProps {
   size?: CardSize;
 }
 
+export const $isCardInFavorites = createStore(false);
+
 export const CardPreview = ({
   card,
-  isCardInFavorite = false,
   href,
   loading = false,
   size = 'small',
 }: CardPreviewProps) => {
+  const favoritesCards = useStore(cardModel.$favoritesCards);
+  const isCardInFavorites = favoritesCards.some((s) => s.id === card.id);
+
+  const addToFavorites = useEvent(cardModel.addedToFavorites);
+  const removeFromFavorites = useEvent(cardModel.removedFromFavorites);
   const historyPush = useEvent(navigationModel.historyPush);
 
+  const handleFavoritesAdd = useCallback((cardId: string) => {
+    addToFavorites({ id: cardId });
+  }, []);
```

Comment:
```suggestion
  const handleFavoritesAdd = useCallback((cardId: string) => addToFavorites(cardId), [addToFavorites]);
```

1. Лучше упрощать передачу данных из вью в модель насколько можно. Так будет проще жить. Лишний объект тут не нужен, если он не разъясняет что именно за строку мы передаем. Мб тут убрать `{ id }` или переименовать в `{ cardId }`?
2. В deps хуков надо передавать вызываемое событие, ведь реакт или эффектор может их пересоздать через `useEvent` из-за деталей реализации, а тут может отвалиться, ищи свищи потом. Лучше перестраховаться, благо это не сложн

Reply:
fixed